src/Illuminate/Support/Facades/Blade.php

```diff
@@ -13,6 +13,7 @@
  * @method static void if(string $name, callable $callback)
  * @method static bool check(string $name, array ...$parameters)
  * @method static void component(string $path, string|null $alias = null)
+ * @method static void prefix(string $prefix)
```

Comment:
Really not obvious this is for components from the name. What about `componentPrefix`?

Reply:
Maybe yeah. We don't use prefixes anywhere else though?